index.html

```diff
@@ -31,5 +32,32 @@ <h3 class="card-title">Real guests reviews</h3>
     </div>
 
 </div>
+
+<footer>
+    <nav class="navigation">
```

Comment:
Please read more about `<nav>` I don't recommend to use `<nav> `for a footer, you should use nav for the main navigation, usually this is in a header or in an aside block
It's better to use nested `<ul>` here
```
<ul>
   <li>
      <h3>Title</h3>
      <ul>...</ul>
  </li>
  ...
</ul>
```